join-statements/joinstatements.md

```diff
@@ -4,63 +4,33 @@
 
-**LEFT OUTER JOIN**- similar to the INNER JOIN, but assuming table A is specified in the `FROM` statement, in the output query, there would be a row for every row in table A. If the conditional statement specified in `ON` is not met, then this row output null values in the columns of table B.
+Our head guidance counselor realizes that in his previous request from the Database department, he was excluding all students that weren't enrolled in a society. These students would also need a guidance counselr. He would like a list of all students, and if they are in a society, list their society too.
 
-Similar to the setup of the INNER JOIN, except if we do a LEFT OUTER JOIN instead, we would get the students already in the extracurricular society and also any student that is not in the extracurricular activity. This could be good in order to see which students could be good recruiting targets.
+**RIGHT OUTER JOIN** - similar to the LEFT INNER JOIN. Lets assume table A is specified in the `FROM` statement, and table B is specified in the `RIGHT OUTER JOIN` statement. In the query, all columns from table B will be represented in the output. Columns pertaining to table A will have values resulting from an `INNER JOIN` if the corresponding conditional statement specified in the `ON` section is TRUE and will be joined with the corresponding row from table B. If the conditional statment specified in the `ON` section is FALSE, columns pertaining to table A will show up as `NULL` values.
 
-```
-SELECT students.name societies.society
-FROM students
-LEFT JOIN societies ON students.id = societies.student_id;
-```
-
-**RIGHT OUTER JOIN** - similar to LEFT OUTER JOIN, except instead of focusing on the table specified in the `FROM` statement (in the previous case it was table A), we would focus on the table linked to the  `RIGHT OUTER JOIN` statement, in this case, table B. Similarly, there will be a single row in the output for every row in table B, and any row that does not meet the conditional in the `ON` statement will have null values in the output values of columns from table A.
-
-In the extracurricular society example shown before, using a RIGHT OUTER JOIN would yield an output similar to the INNER JOIN output, _plus_ if we've got societies with no members, they would show up. This would be useful for example to pinpoint societies for students interested in building a society from the ground up.
-
-```
-SELECT students.name societies.society
-FROM students
-RIGHT JOIN societies ON students.id = societies.student_id;
-```
+The head guidance counselor would like to get a list of all societies, with the students that are part of them, and just show the society name in order to reflect if the society has no students.
```

Comment:
you're still not adding a reason here even though you added to the example

Reply:
done